README.pt.md

```diff
@@ -0,0 +1,84 @@
+# sway
+
+O sway é um compositor do [Wayland](http://wayland.freedesktop.org/) compatível com o [i3](https://i3wm.org/).
+<!-- sway is an [i3](https://i3wm.org/)-compatible [Wayland](http://wayland.freedesktop.org/) compositor. -->
```

Comment:
Maybe remove these?

Reply:
You mean remove the comments?

Reply:
Yeah, is it expected that they remain or were they there just to aid translation?

Reply:
Yes, I just kept those comments to help me with the translation.

Reply:
Should I remove those?

Reply:
Yes, please.

Reply:
I removed them.